### ISSUE TITLE
cant find .env file? 

### ISSUE THREAD

**Issue by Maki9009** [m1guelpf/chatgpt-telegram#31]
for some reason, ti cant finds that file, but its right there in the same directory and its named .env so I'm kinda confused

**Comment by Hormold** [comment]
Check path to directory, if it has spaces, move folder to more short path location.

**Comment by Maki9009** [comment]
@Hormold its in a directory that doesnt have spaces? but ill try it out on my C drive i guess

**Comment by Maki9009** [comment]
yeah its the same issue on my C drive desktop

**Comment by missuo** [comment]
You need to know that, by default, files starting with `.`  file is hidden. You can open it and modify it directly by using the command `vi .env` to create it.

**Comment by Maki9009** [comment]
@missuo but the .env file isn't hidden, I'm using the windows build. when i got the example file, it was called env, i opened it and i added my ID and Token to it, and renamed it to .env and its not working

**Comment by Maki9009** [comment]
Okay i think I fixed that error because for some reason it seems l it also installed straight into the Cdrive and it was calling for that first but now i have a new issue, 
Couldn't load config: Couldn't read config file: While parsing config: invalid character ':' after top-level value
I'm trying to do the cookie method but idk if that's actually the issue, since logging in with google doesn't work

**Comment by ashishbhateja** [comment]
Same issue on the mac. Renamed the file to .env but keep getting file not found error

**Comment by Maki9009** [comment]
same test it out on mac also same issue

**Comment by Maki9009** [comment]
I fixed it, i had to manually rename the .env file in using the terminal, because when you rename normally it doesn't work

**Comment by roa-beep** [comment]
use ur terminal and go to the directory where env.example is then use the following command:

mv env.example .env

**Comment by banghia112** [comment]
@roa-beep i tried renaming it using terminal but i got the same error
<img width="567" alt="image" src="https://user-images.githubusercontent.com/89189266/206638009-7beb5613-7869-483f-9fa8-06b8ce622b32.png">
<img width="782" alt="image" src="https://user-images.githubuse
rcontent.com/89189266/206638036-0c88983e-feca-4c05-8e24-c64250d18e3b.png">

**Comment by RaymondDuke** [comment]
If you're on a mac, you have to first press "cmd + shift + period" to show/hide hidden files.

Once hidden files are shown, you can rename the file to ".env".

It won't let you do it because files that start with "." are protected when hidden files are hidden.

**Comment by abdcef** [comment]
> If you're on a mac, you have to first press "cmd + shift + period" to show/hide hidden files.
> 
> Once hidden files are shown, you can rename the file to ".env".
> 
> It won't let you do it because files that start with "." are protected when hidden files are hidden.

Yes, I have tried it and it is named correctly. The issue persists.

**Comment by m1guelpf** [comment]
Could you see if this works? 
![image](https://user-images.githubusercontent.com/23558090/206866302-67adf8ac-3ade-4ea3-bd40-fbf37b469a0c.png)

**Comment by zweny** [comment]
> Could you see if this works? ![image](https://user-images.githubusercontent.com/23558090/206866302-67adf8ac-3ade-4ea3-bd40-fbf37b469a0c.png)
After modifying the read & write permissions, The “can not find .env " issue persists.

**Comment by VDAODAO** [comment]
Same issue here. Correctly renamed .env file (checked in More info), the program can't find it.

**Comment by kkchaikk** [comment]
This is a really terrible project, and in the face of this bug, it has not been fixed so far.

**Comment by viraj89** [comment]
> Could you see if this works? ![image](https://user-images.githubusercontent.com/23558090/206866302-67adf8ac-3ade-4ea3-bd40-fbf37b469a0c.png)

Not working buddy! 
What next steps from your side on this project? Please intervene and help the community as an owner.

**Comment by qinwanjie** [comment]
same issue

**Comment by qinwanjie** [comment]
if some had fix it,please let me konw

**Comment by dop1e** [comment]
same problem:
config file .env does not exist, using env variables
Invalid .env config: TELEGRAM_TOKEN is not set

**Comment by astroschen** [comment]
window can name the env.example file .env
